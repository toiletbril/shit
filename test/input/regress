(2^2+3*2|5)+(-(42*3^2)-25)+5^2-(52-(2>=2||5>>24)+2-(5+10*2)-5-~5)
