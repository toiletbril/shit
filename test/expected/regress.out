[Binary ^]
  [Binary +]
    [Binary +]
      [Binary |]
        [Binary ^]
          [Const 2]
          [Binary +]
            [Const 2]
            [Binary *]
              [Const 3]
              [Const 2]
        [Const 5]
      [Binary -]
        [Unary -]
          [Binary ^]
            [Binary *]
              [Const 42]
              [Const 3]
            [Const 2]
        [Const 25]
    [Const 5]
  [Binary -]
    [Const 2]
    [Binary -]
      [Binary -]
        [Binary -]
          [Binary +]
            [Binary -]
              [Const 52]
              [Binary ||]
                [Binary >=]
                  [Const 2]
                  [Const 2]
                [Binary >>]
                  [Const 5]
                  [Const 24]
            [Const 2]
          [Binary +]
            [Const 5]
            [Binary *]
              [Const 10]
              [Const 2]
        [Const 5]
      [Unary ~]
        [Const 5]
154
